x-pack/legacy/plugins/infra/public/pages/logs/analysis/setup/process_step/process_step.tsx

```diff
@@ -24,7 +24,8 @@ import { RecreateMLJobsButton } from './recreate_ml_jobs_button';
 
 interface ProcessStepProps {
   cleanupAndSetup: () => void;
-  errorMessages: string[];
+  setupErrorMessages: string[];
+  validationErrors: string[];
```

Comment:
Should this be part of the form validation state displayed in the prior setup step?

Reply:
Yeah good point. I put it here because we had full-width error messages already, but they can be shown above the step title.